### DISCUSSION TITLE
vscode extension support biome configuration

### DISCUSSION THREAD

**Discussion by eightHundreds** [biomejs/biome#1528] [Ideas]
I don't want to add a biome.json to every project. I only want replace prettier vscode extension with biome vscode extension

**Comment by ematipico** [comment]
Disable the option `requireConfiguration`

**Reply by ematipico** [reply]
Maybe I'm wrong. Sorry

**Comment by eightHundreds** [comment]
I mean is the vscode plugin configuration supports configuring biome

**Reply by ematipico** [reply]
We had something like this in the past , but it's a massive endeavour

**Reply by eightHundreds** [reply]
You do not need to declare vscode extension config for each biome configuration item, you can make it to a single json value
